### PR TITLE
Ch6 - generalize concrete test Action Multiply (Array Int)

### PR DESCRIPTION
**Pull request by gomain**
Related to #334, the current test relies on a specific implementation of `Action Multiply Int`.

### CODE REVIEW

File: exercises/chapter6/test/Main.purs [hunk 1]
```diff
@@ -208,7 +208,7 @@ Note to reader: Delete this line to expand comment block -}
             Assert.equal (act m1 (act m2 a))
               $ act (m1 <> m2) a
           test "Multiply Array Int concrete" do
```
Comment (milesfrain) [inline exercises/chapter6/test/Main.purs:210]: Don't know if "concrete" is still the best name here, but can't think of anything better at the moment.

Comment (gomain) [reply exercises/chapter6/test/Main.purs:210]: How about "Acts on array elements"?

Comment (gomain) [reply exercises/chapter6/test/Main.purs:210]: This changes the schema of test-naming though. Maybe do as part of #138.

Comment (milesfrain) [reply exercises/chapter6/test/Main.purs]: > How about "Acts on array elements"?

Sounds good to me

**Comment by milesfrain** [comment]
Superseded by #338

**Pull request closed without merging**